doc/source/getting_started/_notebook_stubs/linear_system.ipynb

```diff
@@ -0,0 +1,613 @@
+       "1.4142135623730951"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "x2.norm()  # sqrt(1^2 + 0^2 + 1^2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "That can be changed, of course:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "6.0"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Z = odl.Rn(3, exponent=1)  # 1-norm\n",
+    "z = Z.element([1, 2, 3])\n",
+    "z.norm()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Setup of data and system matrix"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We use\n",
+    "$$ \n",
+    "A = \n",
+    "\\begin{pmatrix}\n",
+    "  1 & 2 & 0 \\\\\n",
+    "  -1 & 0 & 1\n",
+    "\\end{pmatrix},\n",
+    "\\qquad\n",
+    "b =\n",
+    "\\begin{pmatrix}\n",
+    "  1\\\\\n",
+    "  -1\n",
+    "\\end{pmatrix}.\n",
+    "$$\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "b = Y.element([1, -1])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "A = np.array([[1.0, 2.0, 0.0],\n",
+    "              [-1.0, 0.0, 1.0]])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Make an operator from the matrix"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Mathematical definition:**\n",
+    "\n",
+    "$$\n",
+    "\\mathcal{A}: \\mathscr{X} \\to \\mathscr{Y} \\\\\n",
+    "\\mathcal{A}(x) := Ax\n",
+    "$$\n",
+    "\n",
+    "The `Operator` class is the universal interface to solvers. For matrices, we can use the predefined `MatVecOperator`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "A_op = odl.MatVecOperator(A, dom=X, ran=Y)  # dom and ran can be omitted - then domain and range are inferred"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To evaluate the operator, one simply calls it like a function:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Rn(2).element([3.0, 0.0])"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "x = X.element([1, 1, 1])\n",
+    "A_op(x)"
```

Comment:
perhaps also show of directly calling with a list and mention that it does magic.